packages/react-examples/src/react/ComboBox/docs/ComboBoxBestPractices.md

```diff
@@ -14,3 +14,9 @@ Combobox dropdowns render in their own layer by default to ensure they are not c
 ```js
 calloutProps={{ doNotLayer: true }}
 ```
+
+#### Truncation
+
+By default, the Combobox truncates option text instead of wrapping to a new line. To avoid losing meaningful information, adjusting styles to wrap the name while keeping additional secondary text short is recommended.
```

Comment:
Nit (wordsmithing):  Because this can lose meaningful information, it is recommended to adjust styles to wrap the name. and keep additional secondary text short.